content/configuration/module.md

```diff
@@ -133,6 +133,22 @@ W> This option is __deprecated__ in favor of `Rule.use`.
 
 An array of [`Rules`](#rule) from which only the first matching Rule is used when the Rule matches.
 
+```javascript
+{
+  test: /.css$/,
+
+  oneOf: [
+    {
+      resourceQuery: /inline/, // foo.css?inline
+      use: 'url-loader',
+    },
+    {
+      resourceQuery: /external/, // foo.css?external
+      use: 'file-loader',
+    },
+  ],
```

Comment:
Can you drop the trailing `,` in the example? I know it's valid ES6 but we've tried to stay away from it in the docs for consistency (it also looks a bit cleaner without them).